packages/create-next-app/templates/typescript/tsconfig.json

```diff
@@ -1,18 +1,20 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "moduleResolution": "node",
+    // Allow all features in input.
+    "module": "esnext",
+    // Emit ES2015 over ES5 for smaller bundles.
+    "target": "es2015",
```

Comment:
This does not change what is being emitted as compilation runs using Babel currently. `es5` is the correct output that matches Next.js defaults closely currently.